pyproject.toml

```diff
@@ -16,12 +18,27 @@ classifiers = [
     "Operating System :: OS Independent",
 ]
 dependencies = [
-    "elasticsearch",
-    "tqdm",
-    "pymysql>=1.0",
+    "elasticsearch==8.9.0",
+    "PyMySQL==1.1.0",
+    "pytest==7.3.1",
```

Comment:
Missed this during initial review, can you move pytest to dev time dependencies as well?